service/cluster/kubernetes_cluster.go

```diff
@@ -31,7 +31,6 @@ import (
 
 var operatorStatusesMap = map[k8sclient.OperatorStatus]controllerv1beta1.OperatorsStatus{
```

Comment:
This status is always used with version, so if we don't have `Unsupported` status, then let's remove status at all and use empty `version` as `NotInstalled`

Reply:
done